### ISSUE TITLE
Overlay label shows up behind that card!

### ISSUE THREAD

**Issue by Infinitism** [alexbrillant/react-native-deck-swiper#367]
On swipe action, the overlay label shows behind the card on swipe right or left. It has something to do with the zIndex of the overlay label. What's the way to make it show up on top of the card instead of behind it. When it's behind, it actually cutting off and only "Li" or "pe" show up on right/left swipe of "like" and "nope".

**Comment by VincentLeV** [comment]
I have the same problem but I realize that it's not because of the shadow itself but because of elevation. Only when I use elevation, the label doesn't show

**Comment by kerron** [comment]
I don't think this is going to get fixed, as the last commit was 3+ years ago, and there's currently 122 issues open. First of which is from 2017.

**Comment by webraptor** [comment]
@kerron the package is maintained from https://github.com/webraptor/react-native-deck-swiper for about an year now. Please open issues against that one. PRs welcomed

**Comment by arund-zp** [comment]
Can anyone confirm this issue is for android only?
